src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -336,7 +336,6 @@ const useGroupInfo = () => {
       setIsLoadingInitChat(true);
       await handleAnonymousMessage(selectedUser);
     } catch (e) {
-      console.log(e);
     } finally {
       setIsLoadingInitChat(false);
       setOpenModal(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the catch block, there's no error handling logic. It's important to handle errors appropriately for better debugging and user experience. Consider logging the error or showing a user-friendly message.
```diff
-      } catch (e) {
-      } finally {
+      } catch (e) {
+        console.error(e);
+      } finally {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->